src/hotspot/share/gc/shenandoah/shenandoahHeapRegion.inline.hpp

```diff
@@ -44,43 +44,30 @@ HeapWord* ShenandoahHeapRegion::allocate_aligned(size_t size, ShenandoahAllocReq
 
   HeapWord* orig_top = top();
   size_t addr_as_int = (uintptr_t) orig_top;
```

Comment:
addr_as_int isn't needed anymore, and can be deleted?

Reply:
Thanks.  Removed.